content/docs/intro/vs/cloud_templates.md

```diff
@@ -11,21 +10,25 @@ menu:
 aliases: ["/docs/reference/vs/cloud_templates/"]
 ---
 
+## What is CloudFormation?
+
 All major cloud providers offer their own form of infrastructure-as-"code" solution, typically by way of JSON or
 YAML-based templating solutions. This includes AWS CloudFormation and Azure Resource Manager (ARM) templates.
 
 These markup-based configuration files are often uploaded to a hosted service in the target cloud, where a hosted
 service will then process the files to create, update, or delete resources as necessary.
 
-Pulumi's model shares a lot with these systems. Although Pulumi programs are written in imperative, familiar languages,
-they are ultimately evaluated to produce a similar set of create, update, or delete operations for your cloud resources.
+## The Pulumi Difference
```

Comment:
Same with this one -- the phrase has an advertising-y ring to it and doesn't add much value, and since already seem to rank #1 for "difference between pulumi and cloudformation", presumably because of the "vs" in the title, so I'd suggest dropping this heading, too.

Reply:
👍